src/python_testing/TC_CADMIN_1_5.py

```diff
@@ -128,7 +128,7 @@ async def test_TC_CADMIN_1_5(self):
         sleep(190)
 
         self.step(4)
-        await self.commission_on_network(setup_code=params.commissioningParameters.setupPinCode, discriminator=params.randomDiscriminator)
+        await self.commission_on_network(setup_code=params.commissioningParameters.setupPinCode, discriminator=params.randomDiscriminator, expected_error=0x00000032)
```

Comment:
please add a named constant for 0x32.

Beyond that, test plan says "This may be a failure to find the commissionable node or a failure to establish a PASE connection." ... add a comment explaining why 0x32 is the general code here? Will this throw on success?

Should we instead have a try/catch and treat all errors as ok (or at least all chiperrors as one)?

Reply:
I can add those constant, I based on existing script approach validation